src/ann_solo/reader.py

```diff
@@ -1,11 +1,18 @@
+import io
+import mmap
+from multiprocessing import cpu_count
 import collections
 import logging
-import os
 import pickle
 from functools import lru_cache
+import os
+import re
 from typing import List, Tuple, Dict, IO, Iterator, Union
 
-import joblib
+
+import h5py
+import joblib import Parallel, delayed
+from nltk.tokenize import RegexpTokenizer
```

Comment:
**issue:** Ideally I'd like to avoid the RegexpTokenizer because NLTK is a pretty big additional dependency. Can we achieve the same result with standard regular expressions?